### PR TITLE
Make fraction of managed invokers configurable.

### PR DESCRIPTION
**Pull request by cbickel**
This PR makes the fraction of the pool of managed invokers configurable. This also allows to have an overlap between the pool of blackbox invokers and the pool of managed invokers.

## Related issue and scope
The topic has been discussed here: https://lists.apache.org/thread.html/508e10ec9e800bb239363861385818c95d90ad1aa7df64f3b8904580@%3Cdev.openwhisk.apache.org%3E

## My changes affect the following components

- [ ] API
- [ ] Controller
- [ ] Message Bus (e.g., Kafka)
- [x] Loadbalancer
- [ ] Invoker
- [ ] Intrinsic actions (e.g., sequences, conductors)
- [ ] Data stores (e.g., CouchDB)
- [ ] Tests
- [ ] Deployment
- [ ] CLI
- [ ] General tooling
- [ ] Documentation

## Types of changes

- [ ] Bug fix (generally a non-breaking change which closes an issue).
- [x] Enhancement or new feature (adds new functionality).
- [ ] Breaking change (a bug fix or enhancement which changes existing behavior).

## Checklist:

- [x] I signed an [Apache CLA](https://github.com/apache/incubator-openwhisk/blob/master/CONTRIBUTING.md).
- [x] I reviewed the [style guides](https://github.com/apache/incubator-openwhisk/wiki/Contributing:-Git-guidelines#code-readiness) and followed the recommendations (Travis CI will check :).
- [ ] I added tests to cover my changes.
- [ ] My changes require further changes to the documentation.
- [ ] I updated the documentation where necessary.

### CODE REVIEW

File: core/controller/src/main/resources/reference.conf [hunk 1]
```diff
@@ -6,6 +6,7 @@ whisk {
     use-cluster-bootstrap: false
   }
   loadbalancer {
+    managed-fraction: 90%
```
Comment (markusthoemmes) [inline core/controller/src/main/resources/reference.conf]: This means, that a deployment which adjusts the blackbox-fraction today will have overlapping pools in the future if the managed-fraction is not adjusted?

File: ansible/group_vars/all [hunk 2]
```diff
@@ -69,7 +69,6 @@ controller:
   basePort: 10001
   heap: "{{ controller_heap | default('2g') }}"
   arguments: "{{ controller_arguments | default('') }}"
-  blackboxFraction: "{{ controller_blackbox_fraction | default(0.10) }}"
```
Comment (markusthoemmes) [inline ansible/group_vars/all]: Can we keep this default in for now and also define a default "managedFraction" of 1-blackboxFraction to not run at the risk of configuring overlapping pools in existent deployments that do not define a managedFraction?

File: tests/src/test/scala/org/apache/openwhisk/core/loadBalancer/test/ShardingContainerPoolBalancerTests.scala [hunk 3]
```diff
@@ -96,7 +96,7 @@ class ShardingContainerPoolBalancerTests
     IndexedSeq.fill(count)(new NestedSemaphore[FullyQualifiedEntityName](max))
 
   def lbConfig(blackboxFraction: Double) =
-    ShardingContainerPoolBalancerConfig(blackboxFraction, 1)
+    ShardingContainerPoolBalancerConfig(1.0 - blackboxFraction, blackboxFraction, 1)
```
Comment (markusthoemmes) [inline tests/src/test/scala/org/apache/openwhisk/core/loadBalancer/test/ShardingContainerPoolBalancerTests.scala]: Do we need a test to assert that we correctly handle overlapping pools? I think adding a test setting both values to 1 (which should then yield the same pools) should be sufficient.

File: core/controller/src/main/resources/reference.conf (hunk 1 above)
Comment (cbickel) [reply core/controller/src/main/resources/reference.conf]: After your proposal of keeping the default in `group_vars/all`, the issue doesn't exists anymore here.

**Pull request merged**